app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" in visitor:
+            if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+                raise OutdatedVaccineError("Your vaccine is out of date")
+            if not visitor["wearing_a_mask"]:
+                raise NotWearingMaskError("You should wear mask")

```

Comment:
Code Style: Use descriptive error messages. In this case, it would be better to specify who should wear a mask. For example, 'You should wear a mask.' would be a more descriptive error message.